editions/tw5.com/tiddlers/demonstrations/NestedLists.tid

```diff
@@ -0,0 +1,44 @@
+title: NestedLists
+tags: ListWidget
+created: 20150106180000000
+modified: 20150106180000000
+
+The following sidebar tabs give examples of grouped lists created by nesting.
+
+!! Types Tab
+
+The list in the [[Types|$:/core/ui/MoreSideBar/Types]] tab is generated with this markup:
+
+```
+<$list filter="[!is[system]has[type]each[type]sort[type]]">
```

Comment:
Shall we perhaps make a global wikitext helper macro of that for...

**$:/editions/tw5.com/wikitext-macros**

?

I'm not so fond of these macro names...

```
<<wikitext-example>>
<<wikitext-example-without-html>>
```

Perhaps make them...

```
<<example-html>>
<<example>>
```

?

And add...

```
<<example-code>>
```